### ISSUE TITLE
tactics bug

### ISSUE THREAD

**Issue by Kheriel** [WarEmu/WarBugs#17538]
Hello, I am writing to you today to report a problem with the tactics (slayer), occasionally, they are deactivated, and we are forced to disconnect to reactivate, other ppls of my group had the same issue

**Comment by hammerhammerhammer** [comment]
I can confirm it affects other classes as well.  I have the same issue on my Shadow Warrior.  Anytime I go into a scenario or leave the scenario the tactic no longer is working.  This requires me to remove the tactic and then reapply it.  I believe it is related to zoning.

In my case, the tactic No Quarter should reduce the cast time on Festering Arrow and Eagle Eye by 1 second.  If I go into a scenario the cast times will be normal, as if they are not affected by the tactic.  It shows the wrong cast time in the tool tip as well as when I cast the abilities.  If I remove the tactic and then reapply it it starts working as intended.

**Comment by Kheriel** [comment]
yes, an IB of my group had the problem too, and tried destro side today, had the problem with a tactci on my choppa too, so it's happening for everyones it seems

**Comment by Opeteh** [comment]
I can confirm above. To add more details, that happened for my whole group after we left Aldorf siege.

**Comment by Opeteh** [comment]
![image](https://user-images.githubusercontent.com/69940910/90752239-6e9f4880-e2d7-11ea-80bc-2fab392a478e.png)

**Comment by Kheriel** [comment]
i read some new post about tactics dropping, seems it's happening only on character before lvl 40

**Comment by Kheriel** [comment]
thanks for your work Kaelang on reading all issue and closing double ... lots of work :D

**Comment by CaptainoftheHarem** [comment]
I have had it happen on a CR 40 character (40/6X Zealot). I am not sure when they stopped functioning, but I noticed it after I was killed as a champion in city.

**Comment by CrimsonFenris** [comment]
Reported here as well : https://github.com/WarEmu/WarBugs/issues/17636

**Comment by CrimsonFenris** [comment]
I found a workaround in the meanwhile : switch to another (fully empty) tactic set **before teleporting**, and reequip tactic set after. It should reapply correctly (at least stats are correct with re-equipped tactics).

**Comment by CrimsonFenris** [comment]
Another report here : https://github.com/WarEmu/WarBugs/issues/17745

This issue happens with bolstering + changing zone, yesterday in Gunbad with my r39 Swordmaster :
- Enter Gunbad, noticed changing tactics have no effect after a wipe in Masta Mixa room.
- After relog, enter Mount Gunbad with tactics ; tactics worked **but after going into last boss room** (Gunbad Lab) with tactics on ; **tactics cease to work**.
- After relog, enter Gunbad **and** last boss room with **empty tactic set**, then equip them into boss room : **tactics are now working**.

**Comment by Kruge1976** [comment]
No quarter tactic drops off on SW after a fight. If you drop the tactic and reapply, it works till next comment then seems to stop working again (tested on a 33 SW), but obviously its a bit of a pain to do, and if you get into combat, you can't make the change
